fish_prompt.fish

```diff
@@ -594,14 +591,39 @@ function __bobthefish_prompt_k8s_context -S -d 'Show current Kubernetes context'
             if [ "$key" = 'current-context:' ]
                 set -l context (string trim -c '"\' ' -- $val)
                 [ -z "$context" ]
-                and return
+                and return 1
```

Comment:
what does context `1` mean?

Reply:
It's not the value of the context, it's a non-zero return status of the function meaning that the context is not detected, in this case, the calling function will not use the output of the function as the context and detect the namespace.

Reply:
Right. Sorry, I completely missed that :)